### ISSUE TITLE
[Slider] Focus malfunctions with Radix Slider

### ISSUE THREAD

**Issue by soykje** [adevinta/spark#1471]
### Description

We're actually having issue with `:focus-visible` pseudo-class. Related styles being applied on both focus and focus-visible states (reason seems to be `.focus()` method being called [there](https://github.com/radix-ui/primitives/blob/83a8c13bf66f3d9f17d77caeb187a69eb146930b/packages/react/slider/src/Slider.tsx#L388)).

An issue has been opened on Radix repository ([here](https://github.com/radix-ui/primitives/issues/2397)).

### Steps to reproduce

If you visit https://codesandbox.io/p/sandbox/jovial-sara-4z7nlm and focus with mouse click on thumb, you'll notice that you'll end with `focus-visible` styles. Keyboard focus works just as expected. 

### Spark Component and Version

@spark-ui/slider

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

### Additional Information

As this issue seems a minor one (?), we might stay things as is: having the same styles on `focus` and `focus-visible`. We are quite confident that a local fix, or one from Radix team, will be provided soon :+1: 

A second option would be to remove completely `focus-visible styles` here, but I'm not sure about that.

**Comment by soykje** [comment]
Here is a sandbox environment, where Slider source code has been copied: https://stackblitz.com/edit/stackblitz-starters-7sbyyl?file=components%2FSlider%2Findex.tsx.

After investigations, it appears that the programmatical `.focus()` method is also triggered in the `onChange` method that is defined internally in Radix component, and that we cannot override.
Also, the `onPointerDown` method, which can be overriden, has in its implementation (from Radix) a mechanism that merges default `onPointerDown` (which triggers `.focus()`) with custom one.

**tldr;** It seems that the current Radix implementation won't allow us to override this problematic behaviour (on our side). We could try to reimplement the component from scratch (?) or using another library, but not sure this would be a reasonable cost.

**Comment by soykje** [comment]
@acd02 just came with an interesting proposal: https://stackblitz.com/edit/stackblitz-starters-cr55wa?file=components%2FRadixSlider%2FSlider.tsx,components%2FSlider%2Findex.tsx

Rewriting styles this way could seem "hacky" but would help us achieve our goal :+1:

**Comment by thomas-leguellec** [comment]
@turolopezsanabria @Bibisirius Do you want to do a quick Design Review on this one ? Could be nice to not wait Jeremias an entire week.

**Comment by Bibisirius** [comment]
Review : done 👍 

I don't see any blocking points. Looks great!
